doc/test-framework/scorecard.md

```diff
@@ -79,29 +84,245 @@ API server, indicating that it is modifying resources. This test has a maximum s
 
 #### Owned CRDs Have Resources Listed
 
-This test makes sure that the CRDs listed in the [`owned` CRDs section][owned-crds] of the CSV have a `resources` subsection. If the
+This test makes sure that the CRDs for each CR provided via the `--cr-manifest` flag have a `resources` subsection in the [`owned` CRDs section][owned-crds] of the CSV. If the
 test detects used resources that are not listed in the resources section, it will list them in the suggestions at the end of the test.
-This test has a maximum score equal to the number of CRDs listed in the CSV.
+This test has a maximum score equal to the number of CRs provided via the `--cr-manifest` flag.
 
 #### CRs Have At Least 1 Example
 
-This test checks that the CSV has an [`alm-examples` section][alm-examples] in its metadatas' annotations. This test has a maximum score of 1.
+This test checks that the CSV has an [`alm-examples` section][alm-examples] for each CR provided by the `--cr-manifest` flag in its metadata's annotations. This test has a maximum score
+equal to the number of CRs provided via the `--cr-manifest` flag.
 
 #### Spec Fields With Descriptors
 
-This test verifies that every field in the Custom Resource's spec section has a corresponding descriptor listed in
-the CSV. This test has a maximum score equal to the number of fields in the spec section of your Custom Resource.
+This test verifies that every field in the Custom Resources' spec sections have a corresponding descriptor listed in
+the CSV. This test has a maximum score equal to the total number of fields in the spec sections of each custom resource passed in via the `--cr-manifest` flag.
 
 #### Status Fields With Descriptors
 
-This test verifies that every field in the Custom Resource's status section has a corresponding descriptor listed in
-the CSV. This test has a maximum score equal to the number of fields in the status section of your Custom Resource.
+This test verifies that every field in the Custom Resources' status sections have a corresponding descriptor listed in
+the CSV. This test has a maximum score equal to the total number of fields in the status sections of each custom resource passed in via the `--cr-manifest` flag.
+
+## Extending the Scorecard with Plugins
+
+To allow the scorecard to be further extended and capable of more complex testing as well as allow the community to make their own scorecard tests, a plugin system has been implemented
+for the scorecard. To use it, a user simply needs to add a binary or script to a `scorecard/bin` directory in their operator's root directory that runs the plugin. It is
+recommended to run plugins with a script to allow more configuration from the user and place other required assets in another subdirectory in the `scorecard` directory.
+For instance, `scorecard/assets`. The scorecard runs all plugins from the root `scorecard` directory. Since the scorecard will run all executable files in the scorecard's
```

Comment:
```suggestion
The scorecard runs all plugins from the root `scorecard` directory. Since the scorecard will run all executable files in the scorecard's
```